django_project/feti/admin.py

```diff
@@ -215,3 +216,4 @@ class URLAdmin(admin.ModelAdmin):
 
 admin.site.register(EducationTrainingQualityAssurance)
 admin.site.register(NationalQualificationsFramework)
+admin.site.register(Feedback, admin.ModelAdmin)
```

Comment:
A think we need custom admin, make columns like
title | name | date

and make filter by date